setup-ssl.sh

```diff
@@ -11,11 +11,22 @@ fi
 
 # /etc/ssl/private can't be accessed from within container for some reason
 # (@andrewgodwin says it's something AUFS related)  - taken from https://github.com/orchardup/docker-postgresql/blob/master/Dockerfile
-cp -r /etc/ssl /tmp/ssl-copy/
+
+
+
+if [[ ! -d /configs ]]; then
+  echo "Creating SSL config directory at /configs"
+  mkdir -p /configs
+fi
+if [ ! -f /configs/self-signed-ssl.key -o ! -f /configs/self-signed-ssl.pem ] ; then
+    openssl req -x509 -newkey rsa:4096 -keyout /configs/self-signed-ssl.key -out /configs/self-signed-ssl.pem -days 3650 -nodes -sha256 -subj '/CN=gis'
```

Comment:
Why not stick to the original snakeoil certs if nothing is mounted?

Reply:
Yes, will revert it to that here